src/workerd/api/http.h

```diff
@@ -751,13 +752,18 @@ struct RequestInitializerDict {
   // jsg::Optional<kj::String> priority;
   // TODO(conform): Might support later?
 
+
   JSG_STRUCT(method, headers, body, redirect, fetcher, cf, mode, credentials, cache,
              referrer, referrerPolicy, integrity, signal);
   JSG_STRUCT_TS_OVERRIDE(RequestInit<Cf = CfProperties> {
     headers?: HeadersInit;
     body?: BodyInit | null;
     cf?: Cf;
```

Comment:
hmm, does the TS override here need to account for the difference in the `cache` property definition? I can't recall off the top of my head how the `jsg::Unimplemented cache;` is currently reflected in the generated types.

Reply:
we might want to introduce a test case for that. Is there another TS test case that I can look at that has similar behavior.